### DISCUSSION TITLE
v20.0.11

### DISCUSSION THREAD

**Discussion by Flyingmana** [OpenMage/magento-lts#1700] [Releases]
## v20.0.11 - 2021-06-25

#1567 Fixed incorrect regex in Varien_Object _underscore()
#1544 Prevent backend_type from being changed automatically on attribute save without the admin realizing
#1555 Performance improvement for catalog product images
#1600 Remove "default" themes from 20.x
#1610 Reinstate getters and setters for $_pageHelpUrl which some third party modules still use
#1532 add Possibility to disable global search in backend
#1513 Add redis as a valid option for global/session_save (and activate Cm_RedisSession module by default)
#1499 Set proper HTTP response code on error
#1623 steamClose dosent void this->_streamHandler in php8
#1073 (BC) Removed IE compatibility
#1523 don't redirect to "downloader" (which doesn't exists)

and merged changes from v19.4.14

<hr /><em>This discussion was created from the release <a href='https://github.com/OpenMage/magento-lts/releases/tag/v20.0.11'>v20.0.11</a>.</em>

**Comment by colinmollenhour** [comment]
Thanks for your hard work and dedication to the community!

**Reply by luigifab** [reply]
You have forgotten to update patch version in Mage.php to 11 😃 .

**Reply by Flyingmana** [reply]
argh, thanks for noticing..., was not carefully enough this time

**Comment by luigifab** [comment]
If you are using Redis, don't forgot to update your _local.xml_ with `<session_save>redis</session_save>`.
